contracts/src/v0.8/ccip/test/feeQuoter/FeeQuoter.t.sol

```diff
@@ -2173,11 +2173,11 @@ contract FeeQuoter_KeystoneSetup is FeeQuoterSetup {
     FeeQuoter.TokenPriceFeedUpdate[] memory tokenPriceFeeds = new FeeQuoter.TokenPriceFeedUpdate[](2);
     tokenPriceFeeds[0] = FeeQuoter.TokenPriceFeedUpdate({
       sourceToken: onReportTestToken1,
-      feedConfig: FeeQuoter.TokenPriceFeedConfig({dataFeedAddress: address(0x0), tokenDecimals: 18})
+      feedConfig: FeeQuoter.TokenPriceFeedConfig({dataFeedAddress: address(0x0), tokenDecimals: 18, isEnabled: true})
```

Comment:
Please add a test to validate the fix

Reply:
To validate that it reverts when not enabled? Will do